src/app/core/entity-components/entity-list/entity-list.component.ts

```diff
@@ -215,6 +215,7 @@ export class EntityListComponent<T extends Entity>
       this.entityConstructor,
       this.allEntities
     );
+    this.loadUrlParams(this.activatedRoute.snapshot.queryParams);
```

Comment:
Adding to the previous comment, this should go back to the `ngOnChanges` function.